addons/mail/static/src/messaging/entity/partner/partner.js

```diff
@@ -34,6 +35,10 @@ function PartnerFactory({ Entity }) {
         // Public
         //----------------------------------------------------------------------
 
+        get partner() {
+            return this.env.entities.Partner.get(this.props.partnerLocalId);
+        }
+
```

Comment:
I'm not sure to understand the goal here? The code calling this method should probably just call `this.env.entities.Partner.get` explicitly instead if needed, but it seems to be a remain from moving code from component to entity since it uses `this.props` which is not defined here.

Reply:
Done